### DISCUSSION TITLE
Is there an LSP that works with Node.js versions ≤ 18?

### DISCUSSION THREAD

**Discussion by RigottiG** [LazyVim/LazyVim#3759] [Q&A] [answered]
I have an old project that uses Node.js version 14.15.0, and it seems that Neovim does not work with this version. Is there a way to code in this project without affecting the configuration for the newer versions (≥ 18) that work today?

![image](https://github.com/LazyVim/LazyVim/assets/13246770/c0f1f672-8c0e-44a7-8291-9338a9faf683)

**Comment by adibfirman** [comment, marked as answer]
I think you can use "typescript-language-server" which is "tsserver"
